### PR TITLE
Fix font-family adjustment for the code snippets container

### PR DESCRIPTION
**Pull request by deiteris**
Duplicated font-family property overrides preceding font-family with the variable.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +74,11 @@ npm start
...
 
 ### Running the tests
+
 ```sh
 npm test
 ```
+
+## API components
+
+This component is a part of [API components ecosystem](https://elements.advancedrestclient.com/)
```
Comment (deiteris) [inline README.md:84]: This link seems to be dead btw. Maybe it should be changed or removed?

Comment (jarrodek) [reply README.md:84]: I am removing this line whenever I update a component.

File: src/BaseStyles.js [hunk 2]
```diff
@@ -81,7 +81,6 @@ pre[class*="language-"] {
   color: var(--code-color, black);
   text-shadow: 0 1px white;
   text-shadow: var(--http-code-snippet-code-text-shadow, 0 1px white);
-  font-family: Consolas, Monaco, 'Andale Mono', 'Ubuntu Mono', monospace;
```
Comment (jarrodek) [inline src/BaseStyles.js:84]: We use `--arc-font-code-family`  to define code fonts. This is still the code view so this should start as a variable at least.

**Review by jarrodek — APPROVED** [review]
LGTM

**Pull request merged**